Integrations/AutofocusV2/AutofocusV2.yml

```diff
@@ -969,6 +969,24 @@ script:
     - contextPath: AutoFocus.Domain.WhoisRegistrant
       description: The name of the registrant.
       type: String
+  - arguments:
+    - default: false
+      description: The label of the exported list.
+      isArray: false
+      name: label
+      required: true
+      secret: false
+    deprecated: false
+    description: Gets export list indicators from AutoFocus.
+    execution: false
+    name: autofocus-get-export-list-indicators
+    outputs:
```

Comment:
This is all the data we need here?
@Arsenikr take a look at this pls

Reply:
this is all the info given in the miner... (the other given is the api-key but that is given to the integration itself)